src/components/Tree/Tree.tsx

```diff
@@ -194,87 +141,55 @@ const reducer = (state: TreeState, action: TreeStateAction): TreeState => {
 
 export const Tree = ({
     id,
-    selectedIds,
-    expandedIds,
-    draggable = false,
+    onDrop,
     onSelect,
     onExpand,
-    onDrop,
     children,
-    multiselect = true,
-    baseItemPadding = {},
+    selectedIds,
+    expandedIds,
+    draggable = false,
+    multiselect = false,
+    'data-test-id': dataTestId = 'tree',
```

Comment:
```suggestion
    'data-test-id': dataTestId = 'fondue-tree',
```
Every default `data-test-id` should be prefixed by `fondue-`, comment is also targeting the other files from the tree